samples/cpp/speech_sample/main.cpp

```diff
@@ -654,8 +653,8 @@ int main(int argc, char* argv[]) {
                 }
                 if (!FLAGS_r.empty()) {
                     // print statistical score error
-                    std::string outputName =
-                        (outputs.size() == 0) ? executableNet.output(0).get_any_name() : output_names[next_output];
+                    std::string outputName = (outputs.size() == 0) ? executableNet.output(next_output).get_any_name()
```

Comment:
we are setting this variable 2 times in the code...I would recommend moving this replacement when we are defining outputs vector, but let's do it next time